src/decision/dMCDA.jl

```diff
@@ -45,6 +48,46 @@ function mcda_methods()
     ]
 end
 
+"""
+    mcda_method_names()::Vector{String}
+
+List of MCDA method names.
+"""
+function mcda_method_names()::Vector{String}
+    return getindex.(split.(string.(mcda_methods()), "."), 2)
+end
+
+"""
+    mcda_method_encoding(mcda_name::String)::Union{Int64, Nothing}
+
+Find the scenario encoding of the given mcda method. Throws `ArgumentError` if method not 
+found.
+"""
+function mcda_method_encoding(mcda_name::String)::Int64
+    method_idx = findfirst(mcda_method_names() .== uppercase(mcda_name))
+    if isnothing(method_idx)
+        msg = "Given MCDA method $(mcda_name) is not in list of mcda methods.\n"
+        msg *= "Possible MCDA methods are $(mcda_method_names())"
+        throw(ArgumentError(msg))
+    end
+    return method_idx
+end
+
+"""
+    decision_method_encoding(method_name::String)::Int64
+
+Get the integer encoding for the type of intervention to be used in the scenario dataframe.
+"""
+function decision_method_encoding(method_name::String)::Int64
+    if method_name == "counterfactual"
+        return COUNTERFACTUAL_SCEN_ENCODING
+    elseif method_name == "unguided"
```

Comment:
```suggestion
    method_name = lowercase(method_name)
    if method_name == "counterfactual"
        return COUNTERFACTUAL_SCEN_ENCODING
    elseif method_name == "unguided"
```

Guard against capitalization (or weird mixes)

Reply:
I've modified the suggestion above to use upper case, as that's what the check below uses.